app/(tabs)/mypage.tsx

```
@@ -24,104 +30,107 @@ export default function MyPage() {
-    queryKey: ["user", session?.user?.id],
-    queryFn: async () => {
-      const { data } = await supabase
-        .from("user")
-        .select("*")
-        .eq("id", session?.user?.id)
-        .single();
-      return data;
-    },
+  const { data: profile, isLoading } = useQuery({
+    queryKey: ["profile", session?.user?.id],
+    queryFn: () => getMyProfile(session?.user?.id!),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)